src/redux/features/projects/projectsThunks.ts

```diff
@@ -16,3 +16,17 @@ export const requestProjects = (organizationId: number, locale?: string | null)
     }
   };
 };
+
+export const requestProject = (projectId: number) => async (dispatch: Dispatch, _getState: () => RootState) => {
+  try {
+    const response = await ProjectsService.getProject(projectId);
+    const { error, data } = response;
+    if (data) {
+      dispatch(setProjectAction({ error, projects: [data] }));
+    }
+  } catch (e) {
+    // should not happen, the response above captures any http request errors
```

Comment:
I copy/pasted this from the request above... is this comment needed? Can we just have an empty catch here? Should we just pipe this error into state somewhere?

Reply:
empty catch should be fine i think